src/ui/server/controllers/insurance/business/map-submitted-data/index.ts

```diff
@@ -38,14 +38,11 @@ const mapSubmittedData = (formBody: RequestBody, application: Application): obje
     delete populatedData.registeredOfficeAddress;
   }
 
-  // deletes companies house input field
+  // convert and populate company number and delete the companies house input field
   if (objectHasProperty(populatedData, INPUT)) {
-    delete populatedData[INPUT];
-  }
+    populatedData[COMPANY_NUMBER] = populatedData[INPUT].toString();
```

Comment:
should this not come from the API output?

Reply:
Good shout, reverted this change